doc/lightning-getbalance.7.txt

```diff
@@ -0,0 +1,39 @@
+LIGHTNING-GETBALANCE(7)
+======================
+:doctype: manpage
+
+NAME
+----
+lightning-getbalance - Command showing the total wallet balance.
+
+SYNOPSIS
+--------
+*getbalance* 
+
+DESCRIPTION
+-----------
+The *getbalance* RPC command displays the total funds either on chain or locked on channels.
+
+RETURN VALUE
+------------
+On success an object will be returned with four entries :
+
+- 'on_chain' - The sum of all the wallet utxos, in satoshis.
+
+- 'on_chain_msat' - The sum of all the wallet utxos, in milli satoshis.
+
+- 'on_channels' - The sum of all our satoshis locked in channels, in satoshis.
+
+- 'on_channels_msat' - The sum of all our satoshis locked in channels, in milli satoshis (rounded).
```

Comment:
As mentioned in the comments, we could sum it up for the user: `total` and `total_msat`